perf/bps_tree.cc

```diff
@@ -4,13 +4,19 @@
 #include <inttypes.h>
 #include <time.h>
 #include <random>
+#include <memory>
 
 #include <benchmark/benchmark.h>
 #include <trivia/util.h>
 
+/* A config to enable height-based and first/last/inc/dec benchmarks. */
+#define ENABLE_EXTRA_BENCHMARKS 0
```

Comment:
I'd rather you reduced the data set size instead of disabling some tests. Even better, make the data size dynamically configurable and set it to a smaller value in the `perf_micro` CI workflow.

Reply:
Reduced the data set since the perf pipeline setup is not the primary goal in this patch.